<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>19</y>
      <width>511</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_1">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(255, 101, 55),
</string>
       </property>
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(66, 255, 77)</string>
       </property>
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgba(52, 103, 255, 1)rgb(101, 171, 255)</string>
       </property>
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string> Brightness: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="Slider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionwindow_1"/>
    <addaction name="actionwindow_2"/>
    <addaction name="actionOpen_WIndow_3"/>
   </widget>
   <widget class="QMenu" name="menuClose">
    <property name="title">
     <string>Close</string>
    </property>
    <addaction name="actionwindow_3"/>
    <addaction name="actionwindow_4"/>
    <addaction name="actionClose_Window_3"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuClose"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionwindow_1">
   <property name="text">
    <string>Open Window 1</string>
   </property>
  </action>
  <action name="actionwindow_2">
   <property name="text">
    <string>Open Window 2</string>
   </property>
  </action>
  <action name="actionwindow_3">
   <property name="text">
    <string>Close WIndow 1</string>
   </property>
  </action>
  <action name="actionwindow_4">
   <property name="text">
    <string>Close Window 2</string>
   </property>
  </action>
  <action name="actionClose_Window_3">
   <property name="text">
    <string>Close Window 3</string>
   </property>
  </action>
  <action name="actionOpen_WIndow_3">
   <property name="text">
    <string>Open WIndow 3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
